release/macos/Makefile

```diff
@@ -16,8 +16,25 @@ clean:
 .binary:
 	$(info Building with xgo ...)
 	mkdir -p build
-	xgo -ldflags="-X 'main.githash=$(GITHASH)'" -targets=darwin-10.10/amd64 $(IMPORT_PATH)
-	mv -f trezord-go-darwin-10.10-amd64 build/trezord
+	docker run -it --rm \
+		-v $(IMPORT_PATH):/go/src/github.com/user/go-project \
+		-w /go/src/github.com/user/go-project \
+		-e CGO_ENABLED=1 \
+		docker.elastic.co/beats-dev/golang-crossbuild:1.19-darwin-debian10 \
+		--build-cmd "go build -o release/macos/build/trezord-amd64" \
+		-p "darwin/amd64"
+	docker run -it --rm \
+		-v $(IMPORT_PATH):/go/src/github.com/user/go-project \
+		-w /go/src/github.com/user/go-project \
+		-e CGO_ENABLED=1 \
+		docker.elastic.co/beats-dev/golang-crossbuild:1.19-darwin-arm64-debian10 \
+		--build-cmd "go build -o release/macos/build/trezord-arm64" \
+		-p "darwin/arm64"
+	docker run -it --rm \
```

Comment:
on mac, there is `lipo` for that; but it's not in docker :) but the actual fat binary seems to be simple, and there is a tool that helps with that

this way, we can just make one binary for all macos and not two separate binaries

Reply:
Ah nevermind, the elastic thing has lipo included (through osxcross). So I put it there instead of the weird un-maintaned tool.